cli/stork-publisher-agent/util_test.go

```diff
@@ -0,0 +1,21 @@
+package stork_publisher_agent
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestFloatToQuantizedPrice(t *testing.T) {
+	quantizedPrice := FloatToQuantizedPrice(72147.681412670819)
+	expectedQuantizedPrice := QuantizedPrice("72147681412670819000000")
+	assert.Equal(t, expectedQuantizedPrice, quantizedPrice)
+
+	quantizedPrice = FloatToQuantizedPrice(3.33595034988)
+	expectedQuantizedPrice = "3335950349880000000"
+	assert.Equal(t, expectedQuantizedPrice, quantizedPrice)
+
+	// todo: this case fails
+	//quantizedPrice = FloatToQuantizedPrice(3.335950349883)
```

Comment:
the conversion from float to quantized price introduces some rounding error - this same issue happens in the Rust code.

Might make more sense to take in a string representation of price or allow users to submit already-quantized prices to the agent. Leaving as is for now because price service and Hopper both output floats.

Reply:
Do we need the data to be in terms of float at any point? Maybe we should instead convert everything to "Big" by default